src/store/auth.js

```diff
@@ -48,7 +48,9 @@ const actions = {
       })
       .catch((error) => {
         commit('auth_error')
-        throw error
+        if (error.response.status !== 401) {
```

Comment:
I'm not sure this is equivalent to what happens now as `auth_error` is being comitted when previously it was not. API returning 401 is expected and not to be considered an error.